README.md

```diff
@@ -4,7 +4,7 @@ changes will be needed to adapt how the project operates both from a
 technical standpoint as well as to address social topics such as governance and
 expected conduct.
 
-To address these changes in an open manner, all major changes to the project
+To address these changes in an open manner to the project, it
```

Comment:
it's a little strange...

Reply:
this might be: To address these changes, conda-forge relies on a mechanism called CFEP, which stands for Conda-Forge Enhancement Proposal. These CFEPs
are similar to [Python's PEP](https://www.python.org/dev/peps/) and
[IPython's IPEP](https://github.com/ipython/ipython/wiki/IPEPs:-IPython-Enhancement-Proposals) processes. The CFEP should provide a concise description of the proposed changes as well as it's rationales.

Reply:
And maybe this comment from Matt too: Please consult our [governance docs](https://conda-forge.org/docs/orga/governance.html) and CFEPs 0 and 1 for the scope and purpose of CFEPs.)